src/main/java/org/jenkinsci/test/acceptance/plugins/config_file_provider/ProvidedFile.java

```diff
@@ -57,7 +59,17 @@ public void save() {
     }
 
     public void remove() {
-        visit(url("removeConfig?id=" + this.fileId));
+        // From config-file-provider 3.7.1 on, the remove element is not an anchor anymore.
+        // Use the direct link with versions without the security improvement, otherwise go to the url via GET
+        
+        boolean useOldAnchor = getJenkins().getPlugin("config-file-provider").getVersion().isOlderThan(new VersionNumber("3.7.1"));
+
+        if (useOldAnchor) {
+            WebElement e = find(by.xpath("//a[contains(@onclick, 'removeConfig?id=" + this.fileId + "')]"));
+            e.click();    
+        } else {
+            visit(url("removeConfig?id=" + this.fileId));
```

Comment:
The upstream page is RequirePost so whilst the visit will work it will not remove the config file.

Reply:
This method is not used by any test. I updated it because I bulk changed all links to removeConfig in code. The method doesn't work, I tested it and we get:
![image](https://user-images.githubusercontent.com/22069922/123273168-afbd4600-d502-11eb-88e2-24e28da397a8.png)
So I'm going to remove it from ATH code.